openedx/core/djangoapps/ace_common/templates/ace_common/edx_ace/common/base_body.html

```diff
@@ -159,7 +159,8 @@
                     <tr>
                         <!-- Actions -->
                         <td style="padding-bottom: 20px;">
-                            {% for action_link_url, action_link_text in channel.action_links %}
+                            {% get_action_links channel omit_unsubscribe_link=omit_unsubscribe_link as action_links %}
```

Comment:
What a wonky way to call a function.
Reminder here that if this change is the way we end up implementing this, you need to update every template in edx-platform since it is a breaking change. (hopefully that catches most of them... I'm not sure if other dependencies use the edx-ace version installed by edx-platform.. We would need to do some research into this before we merge)

Reply:
with my new solution it will no longer be a breaking change.
the only other template calling this is base_body.html under red-theme which i can switch